packages/controlled-form/src/lib/Datepicker.tsx

```diff
@@ -40,24 +46,28 @@ export const ControlledDatepicker = ({
         shouldUnregister,
         validate,
         value,
+        ...rules,
       }}
       shouldUnregister={shouldUnregister}
-      render={({ field: { onChange, value }, fieldState: { error } }) => (
+      render={({ field: { onChange, value, onBlur, ref }, fieldState: { error } }) => (
         <Datepicker
           {...rest}
           FieldProps={{
             ...FieldProps,
-            required: typeof required === 'object' ? required.value : !!required,
```

Comment:
Removing this broke the asterisk. It is needed to indicate to the underlying TextField that it is required and to show the asterisk.